pkg/acsengine/template_generator.go

```diff
@@ -1225,5 +1225,15 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 		"shellQuote": func(s string) string {
 			return `'` + strings.Replace(s, `'`, `'\''`, -1) + `'`
 		},
+		"GetPlatformFaultDomainCountForManagedDisks": func() int {
+			switch cs.Location {
+			case "eastus", "eastus2", "westus", "centralus", "northcentralus", "southcentralus", "northeurope", "westeurope":
```

Comment:
According to [the doc](https://github.com/MicrosoftDocs/azure-docs/blob/master/includes/managed-disks-common-fault-domain-region-list.md), "canadacentral" also has 3 max fault domains, so it should be in this list.